p2p/peer.go

```diff
@@ -271,7 +271,6 @@ func (p *peer) send(chID byte, msg proto.Message, sendFunc func(byte, []byte) bo
 	} else if !p.hasChannel(chID) {
 		return false
 	}
-	metricLabelValue := p.mlc.ValueToMetricLabel(msg)
```

Comment:
I wonder whether there was a reason for calling this here, as `msg` can be changed in line 276...

Reply:
Indeed, it breaks everything. Restored.